tests/test_modeling_tf_utils.py

```diff
@@ -359,6 +365,7 @@ def test_special_layer_name_sharding(self):
                 for p1, p2 in zip(model.weights, ref_model.weights):
                     assert np.allclose(p1.numpy(), p2.numpy())
 
+    @require_safetensors
```

Comment:
`safetensors` is now a base dependency so maybe we should eventually just remove all of these

Reply:
Makes sense - do you want me to just do it in this PR?